docs/_tutorials/confirmed-delivery.md

```diff
@@ -218,7 +216,7 @@ Start the `ConfirmedPublish` to send messages to the queue, passing your Solace
 
 ```
 $ ./ConfirmedPublish HOST
-Solace Systems Messaging API Tutorial, Copyright 2008-2015 Solace Systems, Inc.
+Solace Corp Messaging API Tutorial, Copyright 2008-2017 Solace Corp, Inc.
```

Comment:
Solace Messaging API. Not Solace Corp Messaging API.

Reply:
Okay I'll change that